src/components/views/EResource.js

```diff
@@ -10,48 +10,47 @@ import Title from './Title';
 import PCI from './PCI';
 import { resourceClasses } from '../../constants';
 
-export default class EResource extends React.Component {
-  static propTypes = {
-    data: PropTypes.shape({
-      eresource: PropTypes.shape({
-        class: PropTypes.string,
-        name: PropTypes.string,
-        type: PropTypes.object,
-      }),
+const propTypes = {
+  data: PropTypes.shape({
+    eresource: PropTypes.shape({
+      class: PropTypes.string,
+      name: PropTypes.string,
+      type: PropTypes.object,
     }),
-    handlers: PropTypes.shape({
-      onClose: PropTypes.func.isRequired,
-      onEdit: PropTypes.func.isRequired,
-      onNeedMorePackageContents: PropTypes.func.isRequired,
-      onToggleTags: PropTypes.func.isRequired
-    }).isRequired,
-    helperApp: PropTypes.func,
-    isLoading: PropTypes.bool,
-  }
+  }),
+  handlers: PropTypes.shape({
+    onClose: PropTypes.func.isRequired,
+    onEdit: PropTypes.func.isRequired,
+    onNeedMorePackageContents: PropTypes.func.isRequired,
+    onToggleTags: PropTypes.func.isRequired
+  }).isRequired,
+  helperApp: PropTypes.func,
+  isLoading: PropTypes.bool,
+};
 
-  renderEditEresourcePaneMenu = () => {
-    const {
-      data: { eresource = {} },
-      handlers,
-    } = this.props;
+const EResource = ({
+  data: { eresource = {} },
```

Comment:
Always take opportunities like this to clean things up. There's no need to destructure `eresource` and `data` individually. Find the places where you're using `data.eresource` and simplify them.

Reply:
Yeah, so we need both as we are passing the data prop down to the `EResourceViewComponent` component.